lms/static/sass/xblocks/drag_and_drop.scss

```diff
@@ -203,7 +206,19 @@
 }
 
 body.new-theme {
-
+    &.rtl .popup{
```

Comment:
@prakashpathak02 .new-theme comes under the .rtl instead of .rtl comes under the .new-theme. please verify.

Reply:
@shahidrafique new-theme and rtl both are on body. Have used '&' selector

Reply:
but if .rtl comes before .new-theme then your selector will not work.

Reply:
I just checked your code, we are following body.rtl.new-theme only so we expect both classes on body when in RTL

.rtl {
	&.new-theme {}

This is from diagnostic.scss

Reply:
.rtl {
&.new-theme {}

this is fine because .rtl comes first... initially I did the same mistake.